### PR TITLE
add an easy way to get the checksums

### PR DESCRIPTION
**Pull request by muescha**
_If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so._

After making all changes to the cask:
- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ ] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:
- [ ] Named the cask according to the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked there are no [open pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [ ] Checked that the cask was not already refused in [closed issues](https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed).

### CODE REVIEW

File: doc/development/adding_a_cask.md [hunk 1]
```diff
@@ -153,6 +153,16 @@ Example:
   app 'TexmakerMacosxLion/texmaker.app'
   ```
 
+## Calculating Checksums
+
+You can leave the checksum for `sha256` and `checkpoint` empty.
```
Comment (vitorgalvao) [inline doc/development/adding_a_cask.md]: This gives the impression they’ll never have to be filled, which is incorrect.

Please *first* say how to correctly get them (for the appcast, link them to the document), and only then suggest `audit` as an alternative.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: doc/development/adding_a_cask.md (hunk 1 above)
Comment (schlueter) [inline doc/development/adding_a_cask.md]: I believe the shasum cli utility is available by default (or at least once XCode cli tools are installed). It's a one liner for anything: `shasum -a 256 <path/to/file>`.

File: doc/development/adding_a_cask.md [hunk 2]
```diff
@@ -153,6 +153,16 @@ Example:
...
+
+To get the checksums run the audit:
+
+```bash
+brew cask audit my-new-cask --download
```
Comment (schlueter) [inline doc/development/adding_a_cask.md]: I believe the shasum cli utility is available by default (or at least once XCode cli tools are installed), so audit would be overkill. It's a one liner for anything: `shasum -a 256 <path/to/file>`.

**Comment by vitorgalvao** [comment]
Closing for lack of interest.

**Pull request closed without merging**